infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationDiscoveryService.java

```diff
@@ -890,7 +890,7 @@ private String getLocalHost() {
     private Optional<LongTaskTimer.Sample> recordLockAcquire(ClusterRole role) {
         return MeterRegistryProvider.getInstance()
                 .map(registry -> registry.more()
-                        .longTaskTimer("logreplication.lock.duration",
+                        .longTaskTimer("logreplication.lock.duration.nanoseconds",
```

Comment:
It seems that this timer is never stopped.  Am I missing something?

Reply:
The timer in micrometer is managed by TaskTimer.Sample, so here is the point where the timer start, and it will stop in around line 921.